sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1086,6 +1086,14 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("REFRESH TABLE: v2 table") {
+    val t = "testcat.ns1.ns2.tbl"
+    withTable(t) {
+      sql(s"CREATE TABLE $t (id bigint, data string) USING foo")
+      sql(s"REFRESH TABLE $t")
```

Comment:
Any suggestion for a reasonable test here?

`ALTER TABLE` doesn't require `REFRESH TABLE` in the following existing test, correct?
```Scala
      sql(s"CREATE TABLE $t (id int, data string) USING $v2Format")
      sql(s"ALTER TABLE $t DROP COLUMN data")
      val table = getTableMetadata(t) // <- this calls TableCatalog.loadTable
```

Reply:
I think a reasonable test is to create a custom v2 implementation which implements `invalidateTable`.

Reply:
We can implement `InMemoryTable. invalidateTable` and simply set a flag on a global object to indicate it has been called.

Reply:
Thanks for the suggestion! Exactly what I was planning to do. :)